releasing/do-release.sh

```diff
@@ -40,12 +40,15 @@ $PREFIX git checkout go.mod go.sum
 # if there are no valid current credentials.
 $PREFIX docker login
 echo "$VERSION" > VERSION
-$PREFIX docker build -t "fullstorydev/grpcurl:${VERSION}" .
-rm VERSION
+
+# Enable qemu and binfmt support
+$PREFIX docker run --privileged --rm tonistiigi/binfmt:qemu-v6.1.0 --install all
+# Create a new builder instance
+export DOCKER_CLI_EXPERIMENTAL=enabled
+$PREFIX docker buildx create --use --name multiarch-builder
 # push to docker hub, both the given version as a tag and for "latest" tag
-$PREFIX docker push "fullstorydev/grpcurl:${VERSION}"
-$PREFIX docker tag "fullstorydev/grpcurl:${VERSION}" fullstorydev/grpcurl:latest
-$PREFIX docker push fullstorydev/grpcurl:latest
+$PREFIX docker buildx build --platform linux/amd64,linux/s390x --tag fullstorydev/grpcurl:${VERSION} --tag fullstorydev/grpcurl:latest --push --progress plain --no-cache .
```

Comment:
Let's add `linux/arm64`, too, to resolve #234.

BTW, I don't have any experience using multi-platform docker images. Since this statement uses tags like `latest` and `${VERSION}`, I guess the platform does not need to be part of the tag, and that Docker hub (or other image registries) has a separate way of partitioning images for different platforms?

Reply:
> Let's add linux/arm64, too, to resolve #234.

I have added arm64, it should work, however please note that **I do not have access to verify image on arm.** Someone using arm can help us validate. I have verified the images on amd and s390x so far.
Also please note that 'go build' step takes higher time for arm and s390x than amd as of now. Hopefully it will improve in future releases.

> BTW, I don't have any experience using multi-platform docker images. Since this statement uses tags like latest and ${VERSION}, I guess the platform does not need to be part of the tag, and that Docker hub (or other image registries) has a separate way of partitioning images for different platforms?

That's right, docker uses manifest lists for a set of platforms and pulls the correct image based on where it is running. [This ](https://www.docker.com/blog/docker-official-images-now-multi-platform/)describes it well.
I have kept the same image names and tags (latest and $VERSION), this will ensure existing amd users are not affected!

Reply:
When I build and push an arm release, I will see if @pquerna can test an ARM image when I build and push one (that's who requested ARM support in docker).